src/routes/faq/+page.svelte

```diff
@@ -171,8 +141,8 @@
 		I have a high ranking belt in another martial art, can it be transferred to Kendo?
 	</summary>
 	<p>
-		Although many martial arts share similar fundamentals, you will start at a similar introduction
-		level in Kendo. Your training in other martial arts might help you advance faster in Kendo.
+		We start everyone with the same basic training in Kendo to build strong fundamentals. However,
+		prior martial arts experience might help you advance faster in Kendo.
 	</p>
 	<p>
 		If you come from a Kumdo background, you may be considered to hold a comparable rank in Kendo.
```

Comment:
@Archie-G Have you gotten this question before? Is it often from people who already have a high rank in another martial art, or is it from people who have any amount of prior martial arts experience? How often do you get this question? I've never heard someone ask this question before, except maybe for Kumdo